tests/run_all_tests.sh

```diff
@@ -41,6 +41,7 @@ for pytestpath in $PYTESTPATHS; do
         spatialite tastypie-spatialite.db "SELECT InitSpatialMetaData();"
     fi
     
-    ./manage_$type.py test $test_name.tests --traceback
+    echo "./manage_$type.py test $test_name.tests --traceback -t $test_name"
+    ./manage_$type.py test $test_name.tests --traceback -t $test_name
```

Comment:
This was the result of a very confusing bug, where the test discovery was picking up tests in the wrong subdirectories.  -t specifies the top-level directory to look in for tests.  A whole stackoverflow QA on the subject: https://stackoverflow.com/questions/44550805/how-to-disable-djangos-test-discovery/44575787